src/prototype_creep_harvest.js

```diff
@@ -85,5 +85,5 @@ Creep.prototype.spawnCarry = function() {
   if (resourceAtPosition > carryCapacity) {
     Game.rooms[this.memory.base].checkRoleToSpawn('carry', 0, this.memory.routing.targetId, this.memory.routing.targetRoom);
   }
-  this.memory.wait = waitTime;
+  this.memory.wait = Math.max(waitTime, 50);
```

Comment:
I would improve the `wait` time via a PID controller, we have the potential, the differential would be next.

As first step, we could have a fixed value, but this should be in the `config.js`